.circleci/config.yml

```diff
@@ -1061,7 +1061,7 @@ jobs:
           command: mv ./builds-test ./builds
       - run:
           name: Activate Selenium Manager for Firefox
-          command: node_modules/selenium-webdriver/bin/linux/selenium-manager --browser firefox
+          command: yarn
```

Comment:
is this right?

Reply:
Nope, must have somehow escaped out of multicursor when typing